src/components/Card/styled.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+const Image = styled.div`
+  display: block;
+  width: 100%;
+  height: 150px;
```

Comment:
I suggest you use relative units like rem. This helps you achieve a more responsive application. Ideally, you should only use pixels for font sizes, borders, and small stuff.

https://thecssworkshop.com/lessons/relative-units